onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -16,12 +13,19 @@
 namespace onnxruntime {
 namespace openvino_ep {
 
+static std::unique_ptr<GlobalContext> g_global_context;
+
 GlobalContext& BackendManager::GetGlobalContext() {
-  static GlobalContext global_context;
-  return global_context;
+  if (!g_global_context)
```

Comment:
Is this required to be thread safe?

Reply:
It's called by the init function so it's never initialized in a multithreaded way. So I think it is safe.

Reply:
Can we document this here to avoid future questions?

Reply:
Yep, done.